Dockerfile

```diff
@@ -34,8 +36,12 @@ RUN /usr/sbin/a2enmod expires rewrite && \
       ln -sfT /proc/self/fd/1 /var/log/apache2/access.log && \
       ln -sfT /proc/self/fd/1 /var/log/apache2/other_vhosts_access.log
 
-# Bind mount your working copy here
-WORKDIR /twfy
+# Codespaces needs the working directory to be /workspaces/theyworkforyou
+# Some scripts want it in twfy, minimise changes needed for codespaces
+WORKDIR /workspaces/theyworkforyou
+RUN ln -s /workspaces/theyworkforyou /twfy
```

Comment:
There's a part of me that doesn't like having environment-specific things in here, but as long as we want to support choice in development environments it's probably going to be inevitable, or lead to lots of convoluted workarounds.

I'd agree that this is better here than in the scripts, etc, themselves where it's more hidden - as you say, they might support the setting at some point, so just setting that here in a backwards-compatible way seems like a reasonable compromise.